### ISSUE TITLE
`switch` expression line spreading

### ISSUE THREAD

**Issue by loilo** [prettier/plugin-php#1176]
Spreading the expression of a `switch` statement behaves differently than in `if`/`else`/`while` structures in a slightly weird way.

**@prettier/plugin-php v0.11.2**
[Playground link](https://loilo.github.io/prettier-php-playground/#N4IgDgTgpgLjCWUIHkwIPYDsDOIBcok8mMA6vACYwAW+AHAAwA0IMAhgEblW14AsLAK7YoAFU648AMzYAbES2zEA5rKgBFQehhR8M+VBYwIbeLJUBhdAFtrbAAoAJe-hCYsulhxMBjKAGUYAE81VzBsCABaACYQFmgAR0F4aHsTZTs9OQUQYhEIGDS2DLYsgwBfFigKeBh0FDR4LElQNmwYMpFK3MwwQQ68EAAeAH4wajAAHUxpgHpZgAIAUQAPNmswNQWAQWnp+CkFgAoAEgA3JCCAGSxlbcwKADVL54gOdBFHtgh4TjUAOXWUAWADIQQsTmx3DQkK9rrd7k8Xkh3p9vr8OACgQBKBbAablPaYADu1DMwNOFwg8MwdwecNeqKgXx+fyggOswLBEKh2mosMuN1piIZKI+zPRbI5UFx+MwhJmOGJtR81GO50FCPpyLe4pZGKxnNB4Mh0P5EDhQrpSOpjL1ksx7JxeIJRLmi1W602wIAQkSDurZLcff1-OZlNQYLIgv5qPUdBaHYauSbeTCIGH4BGozG4wVYUmnZzZa7FaTyYHg6Hw5Ho7H4wXWY7pcaeWakJns3W8wn9VLnXKFdNsMqYKrK7SQzBO7Xcw3E03k63TXyOzWc-X8wuDUWZS75SBykA)

**Input:**
```php
<?php

// Example A

if ($veryLongAndVeryVerboseVariableName && $anotherVeryLongAndVeryVerboseVariableName) {
}

while ($veryLongAndVeryVerboseVariableName && $anotherVeryLongAndVeryVerboseVariableName) {
}

switch ($veryLongAndVeryVerboseVariableName && $anotherVeryLongAndVeryVerboseVariableName) {
}


// Example B

if ($longButSlightlyShorterVariableName && $anotherSlightlyShorterVariableName) {
}

while ($longButSlightlyShorterVariableName && $anotherSlightlyShorterVariableName) {
}

switch ($longButSlightlyShorterVariableName && $anotherSlightlyShorterVariableName) {
}
```

**Output:**
```php
<?php

// Example A

if (
    $veryLongAndVeryVerboseVariableName &&
    $anotherVeryLongAndVeryVerboseVariableName
) {
}

while (
    $veryLongAndVeryVerboseVariableName &&
    $anotherVeryLongAndVeryVerboseVariableName
) {
}

switch (
    $veryLongAndVeryVerboseVariableName &&
        $anotherVeryLongAndVeryVerboseVariableName
) {
}

// Example B

if (
    $longButSlightlyShorterVariableName &&
    $anotherSlightlyShorterVariableName
) {
}

while (
    $longButSlightlyShorterVariableName &&
    $anotherSlightlyShorterVariableName
) {
}

switch (
    $longButSlightlyShorterVariableName && $anotherSlightlyShorterVariableName
) {
}

```

**Comment by evilebottnawi** [comment]
we need check this for prettier too :smile: Maybe bug exists in prettier too :smile:

**Comment by loilo** [comment]
[It seems.](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEB6VACAogDwIYC2ADgDZwYCCAOlDQJYBmGAFACQBucATgJ4Ay0AOYUoAEwBq3HpK4AjCAGc44vFzp5ZZAHKFyAMj0ZWeKBBgALbjP5CREqTPlKVajdt0BKDMBoBfGjQA7uZ0ZCwcUgJQwmLWjorKquqacDoE+obGphZWkbaxDtxOia4paXBePlD+tFAKgXQwYObhnLxRMfa88c5Jbqm6GAZGJmaWXNYddnFFCS7J7umVfgG16Nj4xGEAQquM4SRC2wCuMADKJHSC5jAkPGfmEFzwE31lg8NZY9wXVzd3DyeL3m-XKy2qq2CoXIbEO0RO50u11u90ezysb0WGRG2XGv2RALRwMxAyW3hWtXqjWaByOp3x-1RQIxpSxQ0yoxyXAZKMB6NerNJFXJ1RAABoQBAiDA6NAFMhQKouBBAgAFVQIeUoPDsCB0UTikCyLh4MAAazg5yIpro0WQMC4xzgEpuBBIAHUQvAFNawHAzprGnR2I0eMhwAp5RLbUpnqqTYICHhkAw8CQlBKAFYKHDbE3my1nXR8W1wFNpjMgbM4M62wRkACKxzMZaQqfTzpA1q4sfDsg0cBIhqIalg7v1FmQAA4AAwSkcJd0mojhkdwWOcQ0AR2b8HjUq1IDwCgAtFA4HBRJfDVw4Du6Lf43hE8m2xXO0oCHR7Y6P3XG7urbtpWMAaOOoiTkgABMEoOngoR1gAwhABBJuGpjnoaxxKAAKhoWrAXAvi+EAA) 😁

**Comment by evilebottnawi** [comment]
Good catch :+1:

**Comment by loilo** [comment]
Haven't found an issue over at their repo so far, going to open one.

**Comment by glen-84** [comment]
This was fixed for JavaScript in v1.19.
